src/foundation.cs

```diff
@@ -8194,6 +8194,26 @@ interface NSInputStream {
 
 	}
 
+	delegate bool NSEnumerateLinguisticTagsEnumerator (NSString tag, NSRange tokenRange, NSRange sentenceRange, ref bool stop);
+
+	[Category]
+	[BaseType (typeof(NSString))]
+	interface NSLinguisticAnalysis {
+		[EditorBrowsable (EditorBrowsableState.Advanced)]
+		[Export ("linguisticTagsInRange:scheme:options:orthography:tokenRanges:")]
+		string[] GetLinguisticTags (NSRange range, NSString scheme, NSLinguisticTaggerOptions options, [NullAllowed] NSOrthography orthography, [NullAllowed] out NSValue[] tokenRanges);
+
+		[Wrap ("GetLinguisticTags (This, range, scheme.GetConstant (), options, orthography, out tokenRanges)")]
+		string[] GetLinguisticTags (NSRange range, NSLinguisticTagScheme scheme, NSLinguisticTaggerOptions options, [NullAllowed] NSOrthography orthography, [NullAllowed] out NSValue[] tokenRanges);
```

Comment:
add unit test for this one
it seems it should return `NSLinguisticTagUnit[]` too